.github/workflows/ci.yml

```diff
@@ -84,3 +84,15 @@ jobs:
       run: cargo install cargo-audit
     - name: Run audit command
       run: cargo audit ${{ env.CARGO_AUDIT_BASE_FLAGS }}
+  credentials-scan:
+    name: Credentials scan
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
```

Comment:
I think `fetch-depth: 0` may prevent the problem from being revealed. I'm concerned about the case where the pull request has multiple commits. This concern comes from the following example outlined in the TruffleHog repository: https://github.com/trufflesecurity/trufflehog?tab=readme-ov-file#shallow-cloning.

Reply:
Fixed